nx/source/services/capsa.c

```diff
@@ -322,6 +350,16 @@ Result capsaGetAlbumCacheEx(CapsAlbumStorage storage, CapsAlbumFileContents cont
     return serviceDispatchInOut(&g_capsaSrv, 8013, in, *cache);
 }
 
+Result capsaGetAlbumEntryFromApplicationAlbumEntryAruid(CapsAlbumEntry *entry, const CapsApplicationAlbumEntry *application_entry, u64 appletResourceUserId) {
+    if (hosversionBefore(2,0,0))
+        return MAKERESULT(Module_Libnx, LibnxError_IncompatSysVer);
+    const struct {
+        CapsApplicationAlbumEntry application_entry;
+        u64 aruid;
+    } in = { *application_entry, appletResourceUserId };
```

Comment:
Since this uses `.in_send_pid`, aruid should be loaded from appletGetAppletResourceUserId() not user-input.